great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -1701,53 +1719,30 @@ def _get_interactive_expect_column_values_to_be_between_chart(
+        max_value_predicate: Union[bool, int]
         predicates: List[Union[bool, int]] = []
         for metric_plot_component in metric_plot_components:
-            if strict_min_predicate and strict_max_predicate:
-                predicates.append(
-                    (
-                        (alt.datum.min_value >= alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value >= alt.datum[metric_plot_component.name])
-                    )
-                    | (
-                        (alt.datum.min_value <= alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value <= alt.datum[metric_plot_component.name])
-                    )
+            if strict_min_predicate:
+                min_value_predicate = (
+                    alt.datum.min_value < alt.datum[metric_plot_component.name]
                 )
-            elif strict_min_predicate:
-                predicates.append(
-                    (
-                        (alt.datum.min_value >= alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value > alt.datum[metric_plot_component.name])
-                    )
-                    | (
-                        (alt.datum.min_value <= alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value < alt.datum[metric_plot_component.name])
-                    )
+            else:
+                min_value_predicate = (
+                    alt.datum.min_value <= alt.datum[metric_plot_component.name]
                 )
-            elif strict_max_predicate:
-                predicates.append(
-                    (
-                        (alt.datum.min_value > alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value >= alt.datum[metric_plot_component.name])
-                    )
-                    | (
-                        (alt.datum.min_value < alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value <= alt.datum[metric_plot_component.name])
-                    )
+
+            if strict_max_predicate:
+                max_value_predicate = (
+                    alt.datum.max_value > alt.datum[metric_plot_component.name]
                 )
             else:
-                predicates.append(
-                    (
-                        (alt.datum.min_value > alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value > alt.datum[metric_plot_component.name])
-                    )
-                    | (
-                        (alt.datum.min_value < alt.datum[metric_plot_component.name])
-                        & (alt.datum.max_value < alt.datum[metric_plot_component.name])
-                    )
+                max_value_predicate = (
+                    alt.datum.max_value >= alt.datum[metric_plot_component.name]
                 )
 
+                predicates.append(min_value_predicate & max_value_predicate)
```

Comment:
This looks to be indented one tab too many?

Reply:
Nice catch, it was in the block with the default case so I didn't see the effect.